packages/server/lib/controllers/runner.ts

```diff
@@ -91,8 +91,9 @@ export const runner = {
     return serveRunner(runnerPkg, config, res)
   },
 
-  handle (req: Request, res: Response) {
-    const pathToFile = getPathToDist('runner-ct', req.params[0])
+  handle (testingType, req: Request, res: Response) {
+    // FIXME: Why is runner-ct being used for all injections? Can anyone comment here as to why this changed?
+    const pathToFile = getPathToDist(testingType === 'e2e' ? 'runner' : 'runner-ct', req.params[0])
```

Comment:
Is this the right path forward or does the cross origin code need to live in `runner-ct`. What are your thoughts @lmiller1990 ?

Reply:
Looking at the runner and runner-ct, it appears there is a bit of a circular dependency between the two.

`runner` imports from `@packages/runner-ct/unified-runner` to attach `window.UnifiedRunner = UnifiedRunner`

where as `runner-ct` is importing styles `'@packages/runner/src/main.scss'` (which are likely the styles mentioned[ in item 3](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/runner/README.md) that need moved to packages/app).

It'd say the biggest differences between the two actually comes down to the webpack configuration used. Looks like the runner-ct's webpack config is a bit more sophisticated:
- [include favicon](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/runner-ct/webpack.config.ts#L85)
- [additional module check](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/runner-ct/webpack.config.ts#L64)
- [is doing chunk naming](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/runner-ct/webpack.config.ts#L75)

I suspect that this could really all be combined &. moved in to `packages/app` and it just hasn't been done yet. Seems like `runner-ct` is probably correct here.

Reply:
I'm thinking this is where the favicon is being loaded: https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/app/index.html#L6

Honestly, I think what seems potentially odd / wonky to me here is that the we seems to be injecting the runner-ct asses in the app [here](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/app/src/main.ts#L33
).

The server has an [ignore list of the ct-assets](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/server/lib/server-base.ts#L45) to proxy because [they were already injected 
](https://github.com/cypress-io/cypress/blob/md-10.0-merge/packages/app/src/runner/injectBundle.ts)

Sorta seems like runner-ct is the full app assets where as the runner assets are the "slimmer" version being injected

Reply:
I wonder in the meantime if it makes sense to move the cross origin injection entry point into `runner-ct` then until we can hopefully combine things in the near future (like in `packages/app`). Then the change here can be removed/reverted